libs/langchain-community/package.json

```diff
@@ -190,6 +192,7 @@
     "node-llama-cpp": "3.1.1",
     "notion-to-md": "^3.1.0",
     "officeparser": "^4.0.4",
+    "openai": "^4.72.0",
```

Comment:
Did we need to add this?

Reply:
@jacoblee93 removed, will use `npm install @langchain/openai` instead.

Reply:
note that for this I had to remove this dependency from the stagehand lib itself. It will only work with stagehand version 1.2.0+